### PR TITLE
UI selector selection : adequacy patch mode

### PR DESCRIPTION
**Pull request by rtei-pw**
adequacy patch mode selection

### CODE REVIEW

**Review by Milos-RTEi — APPROVED** [review]

File: src/libs/antares/study/area/list.cpp [hunk 1]
```diff
@@ -335,7 +335,23 @@ bool saveAreaAdequacyPatchIniFile(const Area& area, const Clob& buffer)
...
+        value = 0;
+        break;
+    case Data::adqmPhysicalAreaOutsideAdqPatch:
+        value = 1;
+        break;
+    case Data::adqmPhysicalAreaInsideAdqPatch:
+        value = 2;
+        break;
+    default:
+        value = 0;
```
Comment (hugo-antoine-rtei) [inline src/libs/antares/study/area/list.cpp]: I received RTE's latest specifications, and the expected default is 1

File: src/libs/antares/study/area/list.cpp [hunk 2]
```diff
@@ -1064,13 +1080,27 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
...
+                        area.adequacyPatchMode = Data::adqmVirtualArea;
+                        break;
+                    case 1:
+                        area.adequacyPatchMode = Data::adqmPhysicalAreaOutsideAdqPatch;
+                        break;
+                    case 2:
+                        area.adequacyPatchMode = Data::adqmPhysicalAreaInsideAdqPatch;
+                        break;
+                    default:
+                        area.adequacyPatchMode = Data::adqmVirtualArea;
```
Comment (hugo-antoine-rtei) [inline src/libs/antares/study/area/list.cpp]: Same here, default should be adqmPhysicalAreaOutsideAdqPatch

File: src/libs/antares/study/area/area.cpp [hunk 3]
```diff
@@ -91,7 +91,7 @@ Area::Area(const AnyString& name, uint nbParallelYears) :
 
 Area::Area(const AnyString& name, const AnyString& id, uint nbParallelYears, uint indx) :
  index(indx),
- bUseAdequacyPatch(false),
+ adequacyPatchMode(Data::adqmVirtualArea),
```
Comment (hugo-antoine-rtei) [inline src/libs/antares/study/area/area.cpp]: the default should be changed to adqmPhysicalAreaOutsideAdqPatch here too

File: src/libs/antares/study/area/list.cpp (hunk 1 above)
Comment (rtei-pw) [reply src/libs/antares/study/area/list.cpp]: ok

File: src/libs/antares/study/area/list.cpp (hunk 2 above)
Comment (rtei-pw) [reply src/libs/antares/study/area/list.cpp]: ok

**Comment by hugo-antoine-rtei** [comment]
damn you're fast

File: src/libs/antares/study/area/area.cpp (hunk 3 above)
Comment (rtei-pw) [reply src/libs/antares/study/area/area.cpp]: ok

**Review by hugo-antoine-rtei — APPROVED** [review]

**Pull request merged**